data/users/__init__.py

```diff
@@ -405,6 +405,12 @@ def is_restricted_user(self, username):
 
         return self.state.is_restricted_user(username)
 
+    def is_superuser(self, username):
+        if not features.SUPER_USERS:
+            return False
```

Comment:
Can you add this check [here](https://github.com/quay/quay/blob/7c6a909130c1f8b65e4957567bdd16c5f4f096e9/data/users/__init__.py#L366) too?

Reply:
@Sunandadadi I didn't add it there because I don't see it added for other functions that also have the same logic.